docs/source/testing/larva/ignores.rst

```diff
@@ -0,0 +1,284 @@
+Ignores
+========
+
+When creating Larva tests, you will find that some constantly changing values will make the tests fail at certain points because there is a difference between the current result of the test and the expected result of the test, for example, the “timestamp”. After all, time changes every second. For solving this kind of issue, Larva has several ways for ignoring parts of messages to compare them. This subsection is intended to name and describe all those ignores within the Larva testing tool.
+
+.. list-table:: Overview
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Options
+     - Description
+   * - ignoreRegularExpressionKey
+     - Replaces all occurrences of regular expression with
+      
+       IGNORE
+   * - ignoreKey 
+     - Replaces all matching occurrences of provided key with
+      
+       IGNORE  
+   * - ignoreContentBetweenKeys 
+     - Replaces all characters found between key1 and key2 with
+      
+       IGNORE
+   * - ignoreKeysAndContentBetweenKeys
+     - Replaces all characters found between key1 and key2, 
+       
```

Comment:
Each table cell has an empty line between its first line and its second line. This is confusing. Can you change this?

Reply:
Another reason to not use a table is that the table still does not fit the screen with on my laptop.